CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ macro(build_spdlog)
   )
 endmacro()
 
-if(NOT spdlog_FOUND OR "${spdlog_VERSION}" VERSION_LESS 1.5.0)
+if(NOT spdlog_FOUND OR "${spdlog_VERSION}" VERSION_LESS 1.8.2)
```

Comment:
I'm honestly not sure if we should do this.  On Windows and macOS, this essentially is a no-op; spdlog never exists there, so we'll always build.  Ubuntu Focal ships with spdlog 1.5.0, so we're currently not building it and relying on the system version instead.  If we make this change, we'll start building it on Ubuntu Focal as well.  That has the upside of being the same version we are using on Windows and macOS, but the downside of not using the system version.  Thoughts?

Reply:
After talking to @cottsay about it, I changed this back to 1.5.0.  That means we'll still use the system package on Ubuntu, which is preferable.